<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatePackage</class>
 <widget class="QDialog" name="updatePackage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:century gothic;
color:white;
}
QLabel{
font-size:16px;
}
#head{
border-bottom-left-radius: 20px;
border-bottom-right-radius: 20px;
background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #FFE53B, stop:0.74 #FF2525);
font-size:24px;
font-weight:bold;
}
QLineEdit{
font-size:16px;
font-weight:bold;
color:black;
background:white;
}
#updatePackage,#frame{
background:#282828;
}
#packageDescriptionEdit{
color:black;
background-color:white;
}
QPushButton {
font-size:15px;
font-weight:bold;
border:none;
border-radius:5px;
background: #ff0303;
}
QPushButton:hover {
background:#49ebff;
color:black;
}
#bookingValidate, #durationValidate, #departureValidate{
font-size:12px;
color:red;
background:transparent;
}
QComboBox{
font-size:16px;
background:#3e3f40;
}

QComboBox QAbstractItemView {
font-size:16px;
background-color:#3e3f40;
color:white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>678</width>
     <height>741</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>676</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>UPDATE PACKAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>707</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>707</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="packageNameLabel">
       <property name="text">
        <string>Package Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="packageNameEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="destinationLabel">
       <property name="text">
        <string>Destinations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="destinationEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="hotelLabel">
       <property name="text">
        <string>Hotel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="transportLabel">
       <property name="text">
        <string>Transport</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="hotelEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="transportEdit">
       <property name="placeholderText">
        <string>if none: Not included</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="departureLabel">
       <property name="text">
        <string>Departure</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="bookingStartsLabel">
       <property name="text">
        <string>Booking</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="bookingStartsEdit">
       <property name="placeholderText">
        <string>XX to XX of month</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="departureEdit">
       <property name="placeholderText">
        <string>XX of month</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="NumOfTravellersLabel">
       <property name="text">
        <string>Number Of Travellers Allowed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="durationLabel">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="durationEdit">
       <property name="placeholderText">
        <string>XX Nights &amp; XX Days</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="NumOfTravellersEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="foodlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>345</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Food </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="foodEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>326</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>if none: Not included</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="packageDescriptionEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>separate detail by comma(',')</string>
    </property>
   </widget>
   <widget class="QLabel" name="packageDescriptionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>565</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Package Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pricePerPersonEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>660</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pricePerPersonLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>635</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Price Per Person</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>621</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="packageLabel">
       <property name="text">
        <string>Select Package</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="packageSelect">
       <property name="placeholderText">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>636</x>
      <y>91</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="pics.qrc">
      <normaloff>:/search.svg</normaloff>:/search.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="bookingValidate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>468</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>follow the format &quot;XX to XX of month&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="departureValidate">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>468</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>follow the format &quot;XX of month&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="durationValidate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>547</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>follow the format &quot;XX Nights &amp; XX Days&quot;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
